core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -265,10 +265,10 @@ class TaskSetManagerSuite extends SparkFunSuite with LocalSparkContext with Logg
     // get chosen before the noPref task
     assert(manager.resourceOffer("exec1", "host1", NODE_LOCAL).get.index == 2)
 
-    // Offer host2, exec3 again, at NODE_LOCAL level: we should choose task 2
+    // Offer host2, exec2, at NODE_LOCAL level: we should choose task 3
```

Comment:
Tasks in TaskSet:
0 List(executor_host1_exec1)
1 List(executor_host2_exec2)
2 List(host1, executor_host2_exec2)
3 List()
I will make changes accordingly.